File: java/Tiket.java
public class Tiket {
    //buat atribur privat
    private String id_tiket;
    private String nama_tiket;
    private int jumlah_tiket;
    private double harga_tiket;

    //buat konstrukteor
    public Tiket(String id_tiket, String nama_tiket, int jumlah_tiket, double harga_tiket){
        this.id_tiket = id_tiket;
        this.nama_tiket = nama_tiket;
        this.jumlah_tiket = jumlah_tiket;
        this.harga_tiket = harga_tiket;
    }

    //getter
    public String getId(){
        return id_tiket;
    }
    public String getNama(){
        return nama_tiket;
    }
    public int getJumlah(){
        return jumlah_tiket;
    }
    public double getHarga(){
        return harga_tiket;
    }

    //setter
    public void setID(String id_tiket){
        this.id_tiket = id_tiket;
    }
    public void setNama(String nama_tiket){
        this.nama_tiket = nama_tiket;
    }
    public void setJumlah(int jumlah_tiket){
        this.jumlah_tiket = jumlah_tiket;
    }
    public void setHarga(double harga_tiket){
        this.harga_tiket = harga_tiket;
    }
    
    //buat nampilin data
    public void ShowData(){
        System.out.println("------------------------");
        System.out.println("ID Tiket   : " + getId());
        System.out.println("Nama Tiket : " + getNama());
        System.out.println("Jumlah     : " + getJumlah());
        System.out.printf("Harga      : %.0f\n", getHarga());
    }
    
}

File: php/Tiket.php
<?php

class Tiket {
    // atribut privat
    private $id_tiket;
    private $nama_tiket;
    private $jumlah_tiket;
    private $harga_tiket;
    private $gambar_tiket;

    // konstruktor
    public function __construct($id_tiket = "", $nama_tiket = "", $jumlah_tiket = 0, $harga_tiket = 0, $gambar_tiket = "") {
        $this->id_tiket = $id_tiket;
        $this->nama_tiket = $nama_tiket;
        $this->jumlah_tiket = $jumlah_tiket;
        $this->harga_tiket = $harga_tiket;
        $this->gambar_tiket = $gambar_tiket;
    }

    // getter
    public function getId() {
        return $this->id_tiket;
    }

    public function getNama() {
        return $this->nama_tiket;
    }

    public function getJumlah() {
        return $this->jumlah_tiket;
    }

    public function getHarga() {
        return $this->harga_tiket;
    }

    public function getGambar() {
        return $this->gambar_tiket;
    }

    // setter
    public function setId($id_tiket) {
        $this->id_tiket = $id_tiket;
    }

    public function setNama($nama_tiket) {
        $this->nama_tiket = $nama_tiket;
    }

    public function setJumlah($jumlah_tiket) {
        $this->jumlah_tiket = $jumlah_tiket;
    }

    public function setHarga($harga_tiket) {
        $this->harga_tiket = $harga_tiket;
    }

    public function setGambar($gambar_tiket) {
        $this->gambar_tiket = $gambar_tiket;
    }

    // buat nampilin data
    public function ShowData() {
        echo "------------------------<br>";
        echo "ID Tiket   : " . $this->getId() . "<br>";
        echo "Nama Tiket : " . $this->getNama() . "<br>";
        echo "Jumlah     : " . $this->getJumlah() . "<br>";
        echo "Harga      : Rp " . number_format($this->getHarga(), 0, ',', '.') . "<br>";
    }
}
